hack/update-crd.sh

```diff
@@ -38,8 +38,7 @@ then
 fi
 
 SCRIPT_ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/.. && pwd)
-
-$CONTROLLER_GEN crd:trivialVersions=true,preserveUnknownFields=false paths=${SCRIPT_ROOT}/pkg/apis/volumesnapshot/v1beta1
+$CONTROLLER_GEN crd:trivialVersions=true,preserveUnknownFields=false paths=${SCRIPT_ROOT}/client/apis/volumesnapshot/v1beta1
```

Comment:
Usually all scripts under hack should be able to run directly under the root.  Since we have to run this one from the client directory, we should detect it early and give user proper instructions.  Otherwise it's going to be confusing.

Reply:
@xing-yang as i have added  in [README](https://github.com/kubernetes-csi/external-snapshotter/commit/d8ff17b8510d8fcf9c95817c851a0ad058d259f9#diff-f50f30f8263ab4ef8a963a53b67c9653L33) as well to run this script we have to run this from client directory , so where else we should mention this, please suggest.

Reply:
What I meant is to add a check in update-crd.sh to programmatically detect that the user is not running from the client directory, and print out a message to inform the user this should be run from the client directory.